explainerdashboard/explainer_plots.py

```diff
@@ -820,7 +820,7 @@ def plotly_dependence_plot(X_col, shap_values, interact_col=None,
 
 def plotly_shap_violin_plot(X_col, shap_values, X_color_col=None, points=False, 
         interaction=False, units="", highlight_index=None, idxs=None, round=3,
-        cats_order=None, max_cat_colors=5):
+        cats_order=None, max_cat_colors=5):  # sourcery no-metrics

```

Comment:
Not sure what this comment is about?

Reply:
thats an artifact from [sourcery](https://sourcery.ai/), my apologies.